combat-trainer.lic

```diff
@@ -2008,7 +2008,7 @@ class TrainerProcess
       appraise(game_state, 'careful')
     when 'App Pouch'
       retreat
-      bput('app my pouch quick', 'You sort', 'Appraise what', 'You cannot appraise')
+      bput("app my #{@gem_pouch_adjective} pouch quick", 'You sort', 'Appraise what', 'You cannot appraise')
```

Comment:
❓ Does this process have access to `@gem_pouch_adjective`? Or does it need to be pulled from settings in the constructor?

Reply:
https://github.com/rpherbig/dr-scripts/blob/878da6a5899a3088fac1205a4c7888d2466625b8/combat-trainer.lic#L406 is in LootProcess, the same as this, it should

Reply:
no, I wrongly thought it was a different process, I will fix this.